Framework/CurveFitting/src/Functions/Bk2BkExpConvPV.cpp

```diff
@@ -75,25 +72,44 @@ double Bk2BkExpConvPV::height() const {
   return height[0];
 }
 
-/** Get peak's FWHM
+/**
+ * Get approximate peak width.
  */
 double Bk2BkExpConvPV::fwhm() const {
-  double sigma2 = this->getParameter("Sigma2");
-  double gamma = this->getParameter("Gamma");
+  // get sigma of Gauss with same FWHM as voigt (H)
   double H, eta;
-  calHandEta(sigma2, gamma, H, eta);
-
-  return mFWHM;
+  calHandEta(getParameter("Sigma2"), getParameter("Gamma"), H, eta);
+  const auto s = H / (2 * sqrt(2 * M_LN2));
+  const auto w0 = expWidth();
+  // Gaussian and B2B exp widths don't add in quadrature. The following
+  // tends to gaussian at large S and at S=0 is equal to the intrinsic width of
+  // the B2B exp (good to <3% for typical params)
+  return w0 * exp(-0.5 * M_LN2 * s / w0) + 2 * sqrt(2 * M_LN2) * s;
```

Comment:
^

Reply:
I have clarified that this is the same Eq. used in` BackToBackExponential `- which I found myself, no reference required!